compiler/bindings/python/test/tools/import_onnx_test.py

```diff
@@ -22,6 +22,84 @@ def run_tool(*argv: str):
 
 
 ONNX_FILE_PATH = os.path.join(os.path.dirname(__file__), "testdata", "LeakyReLU.onnx")
+LARGE_WEIGHTS_ONNX_FILE_PATH = os.path.join(
+    os.path.dirname(__file__), "testdata", "conv.onnx"
+)
+
+
+class ImportOnnxwithExternalizationTest(unittest.TestCase):
+    def setUp(self):
+        with tempfile.NamedTemporaryFile(delete=False) as f:
+            self.outputPath = f.name
+
+    def tearDown(self) -> None:
+        if os.path.exists(self.outputPath):
+            os.unlink(self.outputPath)
+        if os.path.exists("custom_params_file.irpa"):
+            os.unlink("custom_params_file.irpa")
+        if os.path.exists(str(self.outputPath) + "_params.irpa"):
+            os.unlink(str(self.outputPath) + "_params.irpa")
```

Comment:
This code pattern for opening a temporary file, storing the name of that file, and then later deleting it may not work on all platforms.

edit: I ran this on Windows and it worked. Phew.